CHANGELOG.md

```diff
@@ -109,8 +109,11 @@ Adding a new version? You'll need three changes:
   [#3469](https://github.com/Kong/kubernetes-ingress-controller/pull/3469)
 - Added Gateway discovery using Kong Admin API service configured via `--kong-admin-svc`
   which accepts a namespaced name of a headless service which should have
-  Admin API endpoints exposed under a named port called `admin`.
+  Admin API endpoints exposed under a named port called `admin`. Gateway 
+  discovery is only allowed to run with dbless kong gateways, because it is
+  unnecessary to run gateway discovery with DB-backed kong gateways
```

Comment:
I feel the wording here could be slightly improved. I'd drop the second part mentioning "because it is unnecessary".